docs/en/Running-Hadoop-MapReduce-on-Alluxio.md

```diff
@@ -88,6 +88,20 @@ to ensure this jar is on the classpath.
 Note that the jars must be installed again for each update to a new release. On the other hand,
 when the jar is already on every node, then the `-libjars` command line option is not needed.
 
+## Check MapReduce with Alluxio integration (Supports Hadoop 2.X)
+
+Before running MapReduce on Alluxio, you might want to make sure that your configurations has been 
+setup correctly for integrating with Alluxio. The MapReduce integration checker can help you achieve this. 
+
+When you have a running Hadoop cluster (or standalone), you can run the following command in the Alluxio project directory:
+
+```bash
+$ checker/bin/alluxio-checker.sh mapreduce <num maps> 
```

Comment:
@LuQQiu @aaudiber , i view "num maps" is an advanced option. I prefer to have something to start very simply. Can we have a default maps number and leave this to be filled by advanced users? So for someone to start, they can simply use the following
```bash
$ checker/bin/alluxio-checker.sh mapreduce
```

Reply:
Also, they are suggested to have `HADOOP_HOME` set, right? If this is the case, let's also document it here.

Reply:
Yehh, num_maps may not needed to provide better usage feeling. HADOOP_HOME cannot be set,  most of the hadoop client nodes may already have hadoop in their PATH: like what @aaudiber suggested:  [ -f "$(which hadoop)" ] && HADOOP_LOCATION="$(which hadoop)"

Reply:
We can document that the tool will search for `hadoop` in ${PATH} as well as under ${HADOOP_HOME}. For further flexibility, we could allow the user an option to specify the path to the hadoop executable

Reply:
Maybe in the future, after all the checker finished, we can have a separate doc that talks about the details of Integration checker.  
alluxio-checker.sh mapreduce [NUM_MAPS] [HADOOP_EXECUTABLE_LOCATION]  
sometimes this is also confusing.

In most cases, we can find the hadoop executable file through our code. I think it is enough.